lib/valkyrie/persistence/solr/model_converter.rb

```diff
@@ -56,15 +71,25 @@ def add_single_values(attribute_hash)
         end
       end
 
+      # Determines whether or not a field is multivalued
+      # @note this is tied to conventions in the Solr Schema
+      # @see https://github.com/samvera-labs/valkyrie/blob/master/solr/config/schema.xml
+      # @see https://lucene.apache.org/solr/guide/7_4/defining-fields.html#defining-fields
```

Comment:
Do we want to use a URL without a version here?  `https://lucene.apache.org/solr/guide/defining-fields.html#defining-fields` redirects to `https://lucene.apache.org/solr/guide/7_4/defining-fields.html#defining-fields` (though of course that could break in the future).